atc/db/migration/migrations/1551384520_fix_build_private_plan.up.go

```diff
@@ -0,0 +1,76 @@
+package migrations
+
+import (
+	"database/sql"
+	"encoding/json"
+)
+
+func (self *migrations) Up_1550160079() error {
+
+	type build struct {
+		id    int
+		plan  sql.NullString
+		nonce sql.NullString
+	}
+
+	tx, err := self.DB.Begin()
+	if err != nil {
+		return err
+	}
+
+	defer tx.Rollback()
+
+	rows, err := tx.Query("SELECT id, private_plan, nonce FROM builds WHERE private_plan IS NOT NULL")
+	if err != nil {
+		return err
+	}
+
+	builds := []build{}
+	for rows.Next() {
+
+		build := build{}
+		if err = rows.Scan(&build.id, &build.plan, &build.nonce); err != nil {
```

Comment:
(Same goes for the 'down' migration.)

Reply:
heh, yeah that makes sense. I'll batch it.

Reply:
I actually remember this now. I don't think this needs to be batched. The `private_plan` only exists for the duration that the build is running. We query for builds where `private_plan` IS NOT NULL so that shouldn't actually be very many things.

Reply:
Currently there are only 42 builds on wings where this is true.